ui/pages/send/send.constants.js

```diff
@@ -7,8 +7,10 @@ import { addHexPrefix } from '../../../app/scripts/lib/util';
 const MIN_GAS_PRICE_DEC = '0';
 const MIN_GAS_PRICE_HEX = parseInt(MIN_GAS_PRICE_DEC, 10).toString(16);
 const MIN_GAS_LIMIT_DEC = '21000';
+const MAX_GAS_LIMIT_DEC = '7920027';
```

Comment:
Now I'm wondering where this comes from? Sorry for all the questions, just want to make sure this is right :)

Reply:
This again comes from the design cc @jakehaugen

Reply:
Ahh right, I remember that, sounds good 👍🏻